expenses/reports.py

```diff
@@ -78,6 +80,40 @@ class TextFieldOption(Option):
     type: str = attr.ib("text", init=False, repr=False)
 
 
+class ReportItemFormatter:
+    def format_money(self, amount: typing.Union[int, float, decimal.Decimal]) -> str:
+        return format_money(amount)
+
+    def format_category(self, c: Category) -> str:
+        return c
+
+    def format_vendor_link(self, vendor: str) -> str:
+        return vendor
+
+
+class CsvFormatter(ReportItemFormatter):
+    """"Subclass for csv items formatting"""
+
+    def format_money(self, amount: typing.Union[int, float, decimal.Decimal]) -> str:
+        return format_number(amount)
```

Comment:
It might be better to use something that can produce two decimal places for this, and output the correct decimal separator (it could also be configurable).